refactor generic email template

elguitarraverde opened the pull request:
refactor generic email template

![image](https://user-images.githubusercontent.com/2836337/219881931-146bd091-890d-48ac-80f6-eeb8d6dd4eaf.png)

![image](https://user-images.githubusercontent.com/2836337/219881959-d97498e4-edf5-4077-bf6d-843294831ef2.png)

gnovaro reviewed (changes requested):
Looks great but the generic email is not generic anymore if we add data related only for ticket better should be duplicate the generic and implement a custom email view for the ticket feature

elguitarraverde commented:
@gnovaro 
We have added a default template for emails.
We have modified the generic view of the emails so that it uses the default template.
we have created a TicketStateChanged Mailable with its corresponding view.

gnovaro reviewed (dismissed):
Great job!
